home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/cooperations/CooperationApiIT.java

```diff
@@ -124,6 +125,7 @@ void createCooperationInvalidIbanTest() {
             .withMessageContaining("Parameter `iban` is invalid - must meet the rule.");
     }
 
+    @Disabled("Usreo is now to unique")
     @Test
     void createCooperationWithExistingUsreoTest() throws ApiException {
```

Comment:
I think we can remove this test as it became just garbage.

Reply:
👌